app/views/import/fogbugz/new.html.haml

```diff
@@ -0,0 +1,45 @@
+- page_title "Fogbugz Import"
+%h3.page-title
+  %i.fa.fa-bug
+  Import projects from Fogbugz
+%hr
+
+= form_tag callback_import_fogbugz_path, class: 'form-horizontal', multipart: true do
+  %p
+    Follow the steps below to export your Fogbugz project data.
```

Comment:
"export" isn't really appropriate here. I'd go with "Follow the steps below to connect to your Fogbugz account."